lib/scenic/adapters/postgres.rb

```diff
@@ -15,6 +18,14 @@ module Adapters
     #
     # @api extension
     class Postgres
+      # Creates an instance of the Scenic Postgres adapter
+      #
+      # @param connection The database connection the adapter should use. This
+      #   defaults to `ActiveRecord::Base.connection`
+      def initialize(connection = ActiveRecord::Base.connection)
```

Comment:
I'm not sure where / how this class is used, but does it matter that an AR connection is grabbed on initialize, versus sometime later when executing a query?